__tests__/unit/core-forger/manager.test.ts

```diff
@@ -119,8 +119,8 @@ describe("Forger Manager", () => {
             jest.resetAllMocks();
 
             jest.spyOn(Crypto.Slots, "getTimeInMsUntilNextSlot").mockReturnValueOnce(2000);
-            jest.spyOn(Crypto.Slots, "getSlotNumber").mockImplementation(epoch => {
-                return epoch ? 1 : 2;
+            jest.spyOn(Crypto.Slots, "getSlotNumber").mockImplementation(timestamp => {
+                return timestamp === undefined ? 1 : 2;
```

Comment:
The logic got reverted, previously we would return 1 if it was defined and now we would return 1 if it is not defined?

Reply:
Erm, it did. But that doesn't change the outcome. The slots just have to be different.

Reply:
ok